experiment/simulation/index.html

```diff
@@ -1,13 +1,80 @@
 <!DOCTYPE html>
 <html>
-<head>
-	<!-- Add CSS at the head of HTML file -->
-    <link rel="stylesheet" href="./css/main.css">
-</head>
-<body>
+	<head>
+		<!-- Add CSS at the head of HTML file -->
+		<meta charset="utf-8">
+		<meta name="viewport" content="width=device-width, initial-scale=1">
+		<link rel="stylesheet" href="https://cdn.jsdelivr.net/npm/bulma@0.9.1/css/bulma.min.css">
+		<link rel="stylesheet" href="./css/main.css">
+		<script src="https://unpkg.com/mathjs@9.2.0/lib/browser/math.js"></script>
+		<script src="https://cdn.plot.ly/plotly-1.35.2.min.js"></script>
+		<script src='https://cdn.plot.ly/plotly-latest.min.js'></script>
+	</head>
+
+	<body class="page">
 		<!-- Your code goes here-->
+		<div class="columns is-mobile is-centered">
+			<div class="column has-text-centered">
+				<h1 class="title">Simple Harmonic Oscillator</h1>
+			</div>
+		</div>
+
+		<div class="columns is-mobile is-centered">
+			<div class="column has-text-centered">
+				<div class="slidecontainer">
+					<p>INITIAL ANGLE(deg): <span id="demo_dis"></span></p>
+					<input type="range" min="0" max="90" value="50" class="slider" id="displacement">
+				</div>
+
+				<div class="slidecontainer">
+					<p>LENGTH(m): <span id="demo_len"></span></p>
+					<input type="range" min="5" max="30" value="15" class="slider" id="height">
+				</div>
+
+				<div class="slidecontainer">
+					<p>MASS(g): <span id="demo_mass"></span></p>
+					<input type="range" min="400" max="1000" value="600" class="slider" id="mass">
+				</div>
+
+				<div class ="but">
+					<button id="play" class="button">Play</button>
+					<button id="pause" class="button">Pause</button>
+					<button id="restart" class="button">Restart</button>
+				</div>
+			</div>
+		</div>
+
+		<div class="columns is-mobile is-centered">
+			<div class="column has-text-centered">
+				<div class="hero is-small">
+					<div class="hero-body has-text-centered" id="main">
+						<div id="rod" className="rod">
+							<div id="pivot"></div>
+							<div id="ball"></div>
+						</div>
+					</div>
+				</div>
+			</div>
+		</div>
+
+		<div id = "graph">
+			<div class ="graphs">
+				<div class="columns is-centered">
+					<div class="column has-text-centered">
+						<div id="plot" ></div>
+					</div>
+
+					<div class="column has-text-centered">
+						<div id ="plot2"></div>
+					</div>
+				</div>
+			</div>
+		</div>
+
+		<!-- Add JS at the bottom of HTML file --> 
+		<script src="https://cdn.jsdelivr.net/npm/animejs@3.0.1/lib/anime.min.js"></script>
+		<script src="path/to/anime.min.js"></script>
```

Comment:
Why is this line still there?

Reply:
We are using animejs in this experiment.

Reply:
And it is being loaded from 2 locations, one of which is 'path/to/anime.min.js' ??